definitions/divorce/json/CaseField.json

```diff
@@ -3196,5 +3196,13 @@
     "FieldType": "MultiSelectList",
     "FieldTypeParameter": "daOption",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2019",
+    "CaseTypeID": "DIVORCE",
+    "ID": "ApplyForDecreeAbsolute",
+    "Label": "Does the petitioner want to apply for Decree Absolute and finalise the divorce?",
```

Comment:
Can we change this to applicant? Izzy's requested Anna to change it, so..

Reply:
changed